<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomePage</class>
 <widget class="QDialog" name="HomePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="timeInfo">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>30</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="weacherLabel">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>65</y>
     <width>29</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="windInfo">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>87</y>
     <width>120</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>未知</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="windLabel">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>87</y>
     <width>29</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="temperatureLabel">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>110</y>
     <width>29</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="weatherInfo">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>65</y>
     <width>120</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>未知</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="temperatureInfo">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>110</y>
     <width>120</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>未知</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="small">
   <property name="geometry">
    <rect>
     <x>118</x>
     <y>505</y>
     <width>86</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="middle">
   <property name="geometry">
    <rect>
     <x>268</x>
     <y>505</y>
     <width>86</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="big">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>505</y>
     <width>86</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="smalllabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>500</y>
     <width>34</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>文泉驿正黑</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="middlelabel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>500</y>
     <width>34</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="biglabel">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>500</y>
     <width>34</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pickupPushButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>170</y>
     <width>125</width>
     <height>123</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="storagePushButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>170</y>
     <width>125</width>
     <height>123</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="setupPushButton">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>350</y>
     <width>84</width>
     <height>108</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="resgistrationPushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>84</width>
     <height>108</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="blanceQueryPushButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>350</y>
     <width>84</width>
     <height>108</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="rechangePushButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>350</y>
     <width>84</width>
     <height>108</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="useProcessPushButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>350</y>
     <width>84</width>
     <height>108</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="priceCheckPushButton">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>350</y>
     <width>84</width>
     <height>108</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="introductionPushButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>350</y>
     <width>84</width>
     <height>108</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="exceHandPushButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>350</y>
     <width>84</width>
     <height>108</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="cityLabel">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>130</y>
     <width>29</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="cityInfo">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>130</y>
     <width>120</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>未知</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="vegetablelabel">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>500</y>
     <width>34</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="vegetable">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>505</y>
     <width>86</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
